android/src/main/java/com/evollu/react/fcm/FIRLocalMessagingHelper.java

```diff
@@ -84,6 +84,10 @@ public void sendNotification(Bundle bundle) {
                     .setVibrate(new long[]{0, DEFAULT_VIBRATION})
                     .setSound(RingtoneManager.getDefaultUri(RingtoneManager.TYPE_NOTIFICATION))
                     .setExtras(bundle.getBundle("data"));
+            
+            if (bundle.containsKey("ongoing") || bundle.getBoolean("ongoing")) {
```

Comment:
shall we remove the second part in the if condition?
if `bundle.containsKey("ongoing")` is false, `getBoolean` shouldn't be called

Reply:
Oops, maybe I should use `&&` rather than `||`.  Second part in the if is required for the case `{ "ongoing": "false" }` in js. Is it okay?

Reply:
sure